powersimdata/input/input_data.py

```diff
@@ -66,6 +67,10 @@ class InputData:
     def __init__(self, data_loc=None):
         """Constructor."""
         self.data_access = Context.get_data_access(data_loc)
+        if type(self.data_access) != LocalDataAccess:
+            self.local_data_access = LocalDataAccess(server_setup.LOCAL_DIR)
+        else:
+            self.local_data_access = self.data_access
```

Comment:
I believe after the recent update, `self.local_data_access`  is not needed anymore